omod/src/main/webapp/pages/metadata/privileges/privilege.gsp

```diff
@@ -71,8 +71,8 @@
             initialValue : ui.encodeHtmlContent(privilege.privilege)
         ])}
     <% } else{ %>
-        <b>${ui.message("general.name")}:</b> ${privilege.privilege}
-        <input type="hidden" name="privilegeName" value="${privilege.privilege}" />
+        <b>${ui.message("general.name")}:</b> ${ui.encodeHtmlContent(privilege.privilege)}
+        <input type="hidden" name="privilegeName" value="${ui.encodeHtmlContent(privilege.privilege)}" />
```

Comment:
`ui.encodeHtmlAttribute` is better suited here I think

Reply:
@isears Fixed

Reply:
Sorry, should have been a little more clear. `encodeHtmlContent` is fine on line 74, but `encodeHtmlAttribute` is better suited for line 75 (because it's an HTML attribute rather than raw HTML).

Reply:
Got it, fixing now

Reply:
Great, thanks for explaining that @isears definitely makes sense.